web/themes/custom/genlyd_aarhus/templates/form/form--user-register-form.html.twig

```diff
@@ -37,8 +37,12 @@
   },
   "noAccountButton": {
     "title": "I do not have an account" | trans
-    }
+  },
+  "texts": {
+    "passwordTitle": "What about the password?" | trans,
+    "passwordDescription": "You will create a password when you verify your email." | trans,
+    "iHaveAccount": "I have an account" | trans
   }
-%}
+} %}
```

Comment:
It is to avoid having to indent all the lines above.
Do we have a validator for twig?